oci/private/tarball.sh.tpl

```diff
@@ -34,5 +42,7 @@ layers="${LAYERS}" \
         --null-input '.[0] = {"Config": env(config), "RepoTags": "${repo_tags}" | envsubst | split("%") | map(select(. != "")) , "Layers": env(layers) | map( "blobs/" + . + ".tar.gz") }' \
         --output-format json > "${STAGING_DIR}/manifest.json"
 
-# TODO: https://github.com/bazel-contrib/rules_oci/issues/217
-tar -C "${STAGING_DIR}" -cf "${TARBALL_PATH}" manifest.json blobs
+add_to_tar "${STAGING_DIR}/manifest.json" "manifest.json"
```

Comment:
STAGING_DIR is not needed anymore. we can use a `mktemp` for the manifest directly.